### ISSUE TITLE
Customizable access to audio play

### ISSUE THREAD

**Issue by slq32** [QubesOS/qubes-issues#5182]
**The problem you're addressing (if any)**
By default each VM can play audio

**Describe the solution you'd like**
Ability to play sound configurable when creating a VM, or ask user for permission.

**Where is the value to a user, and who might that user be?**
Audio play is a known path for data exfiltration. It can also be annoying.

**Describe alternatives you've considered**
Make audio play access configurable via Qube Settings.

**Additional context**
I could not find anything related while searching the issues. If it's a duplicate please feel free to close/remove this.

**Comment by marmarek** [comment]
You can control each VM volume separately in audio mixer. It should be saved across VM restarts. But indeed you need to start the VM first for that (otherwise it isn't listed).

**Comment by slq32** [comment]
Yes, but some VMs don't need audio in the first place, or maybe the user doesn't want a particular VM to play audio. If an option to control this can be added, it would be nice to have.

**Comment by TNTBOMBOM** [comment]
yeah no option within Qubes manager. but as alternative i use debian or fedora minimal , doesnt have audio stuff installed by default thus i can control which template or clone template will has audio or not.

**Comment by brendanhoar** [comment]
> You can control each VM volume separately in audio mixer. It should be saved across VM restarts.

I do not believe this works (effectively) for disposable VMs.

**Comment by ben-grande** [comment]
Set qube property `audiovm` to empty. It is not available via qube manager though.
